client/src/App.js

```diff
@@ -1,39 +1,30 @@
-import './App.css';
 import React, { useState } from 'react';
-/*import Name from './Components/Header'*/
+import './App.css';
+import HGIB from './Components/Header';
+import DropDown from './Components/DropDown';
+import SearchButton from './Components/button';
+import ResultBox from './Components/ResultBox';
+
 
 function App() {
   const [product, setProduct] = useState();
-
   return (
     <body className="body">
       <form>
-        <h1>What would You like to give back?</h1>
-        <select
-          className="custom-select"
-          id="SelectID"
-          value={product}
-          onChange={e => setProduct(e.target.value)}
-        >
-          <option>Select an Option</option>
-          <option>Plastic Transparent</option>
-          <option>Plastic Black</option>
-          <option>Aluminum</option>
-          <option>Aluminum Contaminated</option>
-          <option>Paper Contaminated</option>
-          <option>Paper Coated</option>
-        </select>
-        <br></br>
+        <HGIB />
```

Comment:
You are returning a `body`Element in line 12.
This is semantically wrong, you are already inside the body of your document (see your index.html in client folder).

Reply:
I don't understand what you mean. So I can just delete the body tag? For some reason I cannot see the index.html, but index.css and index.js